dependencies.gradle

```diff
@@ -12,7 +12,7 @@ ext.versions = [
         androidxArchCore            : '2.2.0',
         androidxBrowser             : '1.7.0',
         androidxCompose             : '1.5.4',
```

Comment:
Can we make it 1.6.0 ?

Reply:
Compose UI 1.6.0 has some large behavior changes. We plan to update, but it will take time.